async-nats/src/jetstream/response.rs

```diff
@@ -51,3 +51,20 @@ pub enum Response<T> {
     Err { error: Error },
     Ok(T),
 }
+
+#[derive(Debug)]
+pub struct DirectError {
```

Comment:
Hold off on specializing this error for the time being? We'll do all of them together instead.